openupgrade_scripts/apriori.py

```diff
@@ -6,6 +6,7 @@
 renamed_modules = {
     # odoo
     "account_facturx": "account_edi_facturx",
+    "sale_coupon": "coupon",
```

Comment:
Well. If I imagine the following use case, i'm not sure it's correct : 

- V13 : ``sale_coupon`` installed.

- openupgraded DB with that code in V14 : ``coupon`` : installed ; ``sale_coupon`` not installed. (it is not autoinstallable)

That is not the expected result. don't you think ?

Reply:
That's why in the pre-migration of `coupon`, I added script to install `sale_coupon`.

Reply:
Ow. Sorry, I didn't saw that part. 

:+1: